fuse_publishers/test/test_path_2d_publisher.cpp

```diff
@@ -31,38 +31,40 @@
  *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  *  POSSIBILITY OF SUCH DAMAGE.
  */
+
+// Workaround ros2/geometry2#242
```

Comment:
I think this comment is in the wrong spot, or maybe the workaround is not needed. The workaround was to include `<tf2_geometry_msgs/tf2_geometry_msgs.hpp>` before `<tf2/utils.h>`, but that's not what's happening here.

Reply:
I ended up reordering them just in case
https://github.com/locusrobotics/fuse/pull/305/commits/166b39a5f8a06ff4e648aef37cd3bea47ea17114